### ISSUE TITLE
[Feature] Option to hide the tray icon

### ISSUE THREAD

**Issue by jcassee** [EasyScreenCast/EasyScreenCast#212]
I rarely record my screen, so having a permanent tray icon feels like clutter. Would you consider adding an option to hide it?

**Comment by tobias47n9e** [comment]
Sure that seems like something useful. Would the icon appear with the counter when starting to record?

**Comment by jcassee** [comment]
Yes, it would be nice to have the indicator (and timer) when it is recording, I think.
